components/brave_wallet_ui/utils/routes-utils.ts

```diff
@@ -218,6 +218,35 @@ export const makeSendRoute = (
   return `${WalletRoutes.Send}?${params.toString()}${SendPageTabHashes.token}`
 }
 
+export const makeSwapRoute = (
+  fromToken: BraveWallet.BlockchainToken,
+  fromAccount: BraveWallet.AccountInfo,
+  toToken?: BraveWallet.BlockchainToken,
+  toAccount?: BraveWallet.AccountInfo
```

Comment:
Let's use an object here for the args to prevent placing the to/from values in the wrong order

Reply:
Updated!